redi-connect-front/src/components/atoms/MediaIcons.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { SFC } from "react";
+import "./MediaIcons.scss";
+import linkedin from "../../assets/images/linkedin.svg";
+import twitter from "../../assets/images/twitter.svg";
+import instagram from "../../assets/images/instagram.svg";
+import medium from "../../assets/images/medium.svg";
+
+const SocialMediaIcons = () => {
+  const icons = [
```

Comment:
Let's move this up and out of the component. The component can then be made even lighter along the lines of ...
```
const SocialMediaIcons = () => (
  <div> ... </div>
)
```

Reply:
Done